modules/json/test/src/smithy4s/json/SchemaVisitorJCodecPropertyTests.scala

```diff
@@ -19,19 +19,19 @@ package json
 
 import cats.Show
 import com.github.plokhotnyuk.jsoniter_scala.core._
+import munit._
```

Comment:
Putting this comment on a random line, but I think we can add the new primitive types to https://github.com/disneystreaming/smithy4s/blob/f0b1e647e077f411438034601e12d45f6ab41999/modules/scalacheck/src/smithy4s/scalacheck/SchemaGenerator.scala#L49 and that will make it so these property tests take them into consideration